checks/src/main/java/tests/ThrowsFinally.java

```diff
@@ -0,0 +1,13 @@
+package main.java.tests;
```

Comment:
RSPEC-2176: Class has same name as parent
<!-- lookout footnote separator -->
_Comment made by the analyzer sonarcheck_analyzer._